ui/apps/platform/eslint-plugins/pluginPatternFly.js

```diff
@@ -147,6 +147,68 @@ const rules = {
             };
         },
     },
+    'version-variable-class': {
+        // Require consistent version of PatternFly variable or class.
+        meta: {
+            type: 'problem',
+            docs: {
+                description: 'Require consistent version of PatternFly variable or class',
+            },
+            schema: [],
+        },
+        create(context) {
+            const findErrorMessage = (value) => {
+                const versionExpected = '5';
```

Comment:
Thoughts on pulling this from our package.json and future proofing it for eternity (or longer)?

Reply:
Aha, I forget what is the order of steps during major upgrade:
* Apply codemod to source files.
* Upgrade dependencies.

Reply:
As far as I know, they are mutually dependent and need to go into the same commit. If the order matters when performing both steps, I am not aware of it.

Reply:
PatternFly upgrade is a big commitment (pardon pun),